docker/ubuntu/baseimage/bootstrap.sh

```diff
@@ -36,3 +36,19 @@ apt-get install -y \
 
 # --upgrade to prevent urllib3 errors
 pip3 install transifex-client --upgrade
+
+# compile custom xgettext with newline patch, cf. https://github.com/koreader/koreader/pull/5238#issuecomment-523794831
+# upstream bug https://savannah.gnu.org/bugs/index.php?56794
+GETTEXT_VER=0.20.1
```

Comment:
@NiLuJe Any comments? In the maintainability/beauty sense; it works fine. ;-)

Reply:
I'd possible replace the push/popd block with a subshell, but that's it, lgtm ;).

Reply:
What do you mean, to run it simultaneously with apt-get or something? Image build time isn't a terribly large concern, image pull time matters a lot more.

Reply:
Nope, I meant something like

```shell
(
    cd blah
    <do stuff>
)
```

Reply:
Instead of

```shell
pushd blah {
    <do stuff>
} && popd blah
```

Reply:
You still want to do cd blah || exit in case it fails but sure, that works too. Slightly quicker to type and slightly less Bash i suppose. :-)